src/agent/rustjail/src/seccomp.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 Sony Group Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use oci::LinuxSeccomp;
+
+#[cfg(feature = "seccomp")]
+use libseccomp::*;
+#[cfg(feature = "seccomp")]
+use std::str::FromStr;
+
+#[cfg(feature = "seccomp")]
+pub fn get_filter_attr_from_flag(flag: &str) -> Result<ScmpFilterAttr> {
+    match flag {
+        "SECCOMP_FILTER_FLAG_TSYNC" => Ok(ScmpFilterAttr::CtlTsync),
+        "SECCOMP_FILTER_FLAG_LOG" => Ok(ScmpFilterAttr::CtlLog),
+        "SECCOMP_FILTER_FLAG_SPEC_ALLOW" => Ok(ScmpFilterAttr::CtlSsb),
+        _ => Err(anyhow!("Invalid seccomp flag")),
+    }
+}
+
+// init_seccomp creates a seccomp filter and loads it for the current process
+// including all the child processes.
+#[cfg(feature = "seccomp")]
```

Comment:
You can do this at module level when importing.
```rust
#[cfg(feature = "seccomp")]
pub mod seccomp;
```

This way you can remove all feature toggles from this file and have it cleaner.

Reply:
That makes sense.
I fixed it.